src/python/create.py

```diff
@@ -532,4 +580,4 @@ def _get_relation(related_objs, cell_value, parsed_cell):
 
 def cellToBool(val):
     """Converts a cell value to a useable boolean"""
-    return True if str(val).lower() == "true" else False
+    return True if str(val).lower() != "false" else False
```

Comment:
I am confused, I saw this line, written a bit differently in a other PR.
Anyways, I suggest something clearer (not sure Python allows it, or.
```suggestion
    return str(val).lower() != "false"
```